helpdesk/email.py

```diff
@@ -1090,9 +1090,31 @@ def extract_email_metadata(
                 else DeleteIgnoredTicketException()
             )
 
+    # First try to get ticket ID from the current queue's slug
     ticket_id: typing.Optional[int] = get_ticket_id_from_subject_slug(
         queue.slug, subject, logger
     )
+
+    # If no ticket ID found in the current queue, check all other queues
+    if ticket_id is None:
+        # Get all enabled queues except the current one
+        other_queues = Queue.objects.exclude(id=queue.id).filter(
+            email_box_type__isnull=False, allow_email_submission=True
```

Comment:
A queue may accept email replies from a mailbox but not submissions. 
```suggestion
            email_box_type__isnull=False
```

Reply:
The changes allow matching ticket IDs across all queues with email configured, regardless of their 'allow_email_submission' setting. This is critical because a queue may need to accept email replies to existing tickets even if it doesn't accept new ticket submissions via email.

By storing the original queue and reverting back to it when no existing ticket is found, we ensure new tickets are always created in the appropriate receiving queue while still allowing updates to existing tickets in any queue.

This addresses the common workflow where certain queues only accept updates to existing tickets but not new submissions.